xbmc/video/VideoInfoTag.cpp

```diff
@@ -604,6 +649,13 @@ void CVideoInfoTag::ToSortable(SortItem& sortable, Field field) const
   }
 }
 
+const CRating CVideoInfoTag::GetRating(std::string type) const
+{
+  if (type.empty())
+    return m_ratings.find(m_strDefaultRating)->second;
+  return m_ratings.find(type)->second;
```

Comment:
You need to handle the case where the call to `find` fails or it will go sideways when someone tries to get a rating for a non-existing type.